### PR TITLE
[PLANNER-2055] Random fail test

### PR DESCRIPTION
**Pull request by desmax74**
@wmedvede @rsynek

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello Max, in the similar PR I did for ignoring two kie-server tests I was suggested by QE to use this:

@Category({UnstableOnJenkinsPrBuilder.class})

 instead if the @Ignore. It looks like the style being used by QE to facilitate the tests to still run under certain execution scenarios.
see https://github.com/kiegroup/droolsjbpm-integration/pull/2161#issuecomment-660195071

Probably you can use this strategy here too.

**Comment by rsynek** [comment]
> Hello Max, in the similar PR I did for ignoring two kie-server tests I was suggested by QE to use this:
> 
> @category({UnstableOnJenkinsPrBuilder.class})
> 
> instead if the @ignore. It looks like the style being used by QE to facilitate the tests to still run under certain execution scenarios.
> see [#2161 (comment)](https://github.com/kiegroup/droolsjbpm-integration/pull/2161#issuecomment-660195071)
> 
> Probably you can use this strategy here too.

I am afraid it wouldn't work here - the category is specific for KIE server tests. Either we use @ignore or we need to configure the surefire maven plugin in similar way as failsafe plugin is configured in the kie-server module. See:
https://github.com/kiegroup/droolsjbpm-integration/pull/1418

File: drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java [hunk 1]
```diff
@@ -34,6 +35,7 @@
 
     private Logger logger = LoggerFactory.getLogger(PodAsReplicaTest.class);
 
+    @Ignore //see https://github.com/kiegroup/droolsjbpm-integration/pull/2156
```
Comment (rsynek) [inline drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java:38]: Can we have here a link to Jira about enabling and stabilizing the test, please?

Comment (desmax74) [reply drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java:38]: The test locally is always green, it start a kafka embedded instance, running on other test under heavy load could spent more that 30 sec and a timeout exception is throw, can I see the stack trace of the error ?

File: drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java [hunk 2]
```diff
@@ -34,7 +35,7 @@
 
     private Logger logger = LoggerFactory.getLogger(PodAsReplicaTest.class);
 
-    @Test(timeout = 30000L)
+    @Test(timeout = 600000L) //see random failure https://github.com/kiegroup/droolsjbpm-integration/pull/2156
```
Comment (rsynek) [inline drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java:38]: We probably don't need the link to the PR where it failed anymore; if it fails again despite the increased timeout, we will notice.

**Comment by mareknovotny** [comment]
is this PR still needed? @rsynek @desmax74

**Comment by rsynek** [comment]
@mareknovotny I have no idea if the test has been failing lately. If there is no problem with it any longer, this PR can be closed.

**Comment by mareknovotny** [comment]
@rsynek i haven't seen anything like failing of processOneSentMessageAsLeaderAndThenReplicaTest test so i will wait for @desmax74 but there is no reason to keep it right now IMHO

**Pull request closed without merging**